hack/istio/multicluster/start-minikube.sh

```diff
@@ -89,7 +89,6 @@ openssl genrsa -out "${KEYCLOAK_CERTS_DIR}"/key.pem 2048
 # create certificate signing request
 openssl req -new -key "${KEYCLOAK_CERTS_DIR}"/key.pem -out "${KEYCLOAK_CERTS_DIR}"/csr.pem \
   -subj "/CN=kube-ca" \
-  -addext "subjectAltName = DNS:${KEYCLOAK_HOSTNAME}" \
```

Comment:
This was [added 2 years ago](https://github.com/kiali/kiali/pull/5735/files#diff-c13b81f0fb784c17857c1eba4eb0e368b377bf281a7aa3e55824b58b9fe4f6eaR95)  by @nrfox - I would ask that he check this to make sure this isn't needed for his platform (Ubuntu). I'm also on F41 and this change is OK with me - this command worked.

Reply:
nudge @nrfox

Reply:
*Wakes up from hibernation... LGTM